VideoLocker/src/main/java/org/edx/mobile/util/AppConstants.java

```diff
@@ -1,12 +1,17 @@
 package org.edx.mobile.util;
 
 
-public class AppConstants {
+public final class AppConstants {
```

Comment:
My suggestion would be to make this class final and static (unless ther eis a good reason not to)